package.json

```diff
@@ -11,7 +25,8 @@
     "build": "npm run clean && npm run build-babel",
     "devstart": "nodemon --exec babel-node src/index.js",
     "lint": "eslint --fix ./src/**/*.js",
-    "test": "nyc mocha --require @babel/register --require @babel/polyfill --timeout 1000 --exit"
+    "test": "cross-env NODE_ENV=test nyc mocha --require @babel/register --require @babel/polyfill --exit",
```

Comment:
I just tried testing without cross-env, but I didn't experience any issues

Reply:
It was not work from my own end when i run `npm test` without cross-env. I reach out to @meetKazuki in regard and the result is still the same.
Please advice

Reply:
![error](https://user-images.githubusercontent.com/28379439/63279750-792c9980-c2a1-11e9-9b58-e085815ffa9f.png)

Reply:
Try running it without the ```NODE_ENV=test``` prefix